hack/e2e.sh

```diff
@@ -316,9 +326,8 @@ fi
 
 # legacy path
 go run $ROOT/hack/e2e.go -- "${kubetest_args[@]}" \
-    --deployment "$DEPLOYMENT" \
+    --deployment="$DEPLOYMENT" \
     --up \
     --down \
-    --test-cmd bash \
-    --test-cmd-args="$ROOT/hack/run-e2e.sh" \
-    "$@"
+    --test-cmd="$ROOT/hack/run-e2e.sh" \
+    "${@:-}"
```

Comment:
what the difference between this and before?

Reply:
before `bash` was the test command, now it's run-e2e.sh and instead more args come to it from https://github.com/kubernetes/test-infra/pull/24241